guillotina_elasticsearch/parser.py

```diff
@@ -188,15 +187,11 @@ def process_query_level(params):
     name='default')
 class Parser(BaseParser):
 
-    def __init__(self, request, context):
-        self.request = request
-        self.context = context
-
     def __call__(self, params: typing.Dict) -> ParsedQueryInfo:
         query_info = super().__call__(params)
 
         query = {
-            'stored_fields': SEARCH_DATA_FIELDS,
+            # 'stored_fields': SEARCH_DATA_FIELDS,
```

Comment:
fwiw, stored fields support is nice to have

Reply:
Thouhgt I disabled it because on the original one, refers mostly to fields related to guillotina_cms, and seemed to me that they are acting a bit unexpected... anyways, what's the benefit of having them? I tryied to look for, and what I found is:

`The stored_fields parameter is about fields that are explicitly marked as stored in the mapping, which is off by default and generally not recommended. Use source filtering instead to select subsets of the original source document to be returned.`

That makes me feel like there is something I'm missing... :/

Reply:
It's about optimizing data retrieval from elasticsearch.

Think of stored fields as data stored in reverse indexes in elasticsearch.

If your search result request only requires fields that are already in stored fields, you don't need to pull the source doc from elasticsearch.

Sometimes the source doc has a lot of extra data and pulling it from cold disk can be slow.

Reply:
cool, but this have to be in some way dynamic... (the problem I found it's that it's just a list of fields, mostly from cms related things)

Reply:
I think the easier way to fix this is removing from `SEARCH_DATA_FIELDS` guillotina_cms fields and only keep the default guillotina (core) fields. In a future iteration, we can add a way to customize the list. Sounds good @vangheem @jordic ?

Reply:
Here is my suggestion: https://github.com/plone/guillotina_elasticsearch/pull/66/commits/55986fd0993a5e23d4a823ef05812eb5953ac757 but i can change it if you want

Reply:
Yes, I'm completely fine with it.